tests/functional/scenario1/DatadirScenarioTest.php

```diff
@@ -272,6 +273,21 @@ public function testUser1HaveCorrectAccessAfterProvisioning(): void
         $rows = $user1connection->fetchAll('SELECT * FROM read_schema_table');
         $this->assertCount(3, $rows);
 
+        $queries = $user1connection->fetchAll(
+            '
+                SELECT 
+                    QUERY_TEXT, QUERY_TAG 
+                FROM 
+                    TABLE(INFORMATION_SCHEMA.QUERY_HISTORY_BY_SESSION())
+                WHERE QUERY_TEXT = \'SELECT CURRENT_ROLE() AS "name"\'  
+                ORDER BY START_TIME DESC 
+                LIMIT 1
+            '
+        );
+
+        $runId = sprintf('{"runId":"%s"}', getenv('KBC_RUNID'));
+        $this->assertEquals($runId, $queries[0]['QUERY_TAG']);
```

Comment:
Skusal som to upravit tak ako si pisal ale stale sa mi to nedari, tam bude podla mna nieco s opravneniami toho usera co ma `$user1connection` skusal som tiez `$masterConnection` a to `$queries` mam prazdne. Ale ked sa pozriem na snowflake na tie query tak su otagovane tak by to malo nieco najst. Nenapada ta ako to vyriesit ? Mozno nastavit ine prava na toho testovaneho usera ak by to bolo opravnenim ale to zas neni asi moc dobre.

Reply:
IMHO je správně, že tam žádné runid není. To runid by mělo být v tom, když se vytváří ty schémata, uživatele, atd. Což je ve chvíli, kdy se na ř. 253 volá ta komponenta. Jenže tam se to volá skrz Symfony process a [musel bys tam ten env přidat](https://symfony.com/doc/current/components/process.html#setting-environment-variables-for-processes). Takže za mě je správně, pokud tam ty tagy nejsou. 

Jinak určitě tohle musíš volat na masterConnection - protože jen uživatel vidí sám svoji query history.

Reply:
Myslis to potom tak ze tu ten test ma byt alebo nema a mam ten env este nejak predat tomu symfony processu pretoze potom mi pride divne ze ked si to pustim lokalne kde si nastavim to run id rucne a spustim testy a potom sa pozriem na snowflake ta tam tie query otagovane vidim a zrovna ta `SELECT CURRENT_ROLE() ...` mi prisla ze sa vola ked sa spusta ta komponenta